app/services/metrics_service.rb

```diff
@@ -13,9 +13,19 @@ def fetch_time_series(base_path:, date_range:)
     api.client.get_json(url).to_hash
   end
 
+  def content_summary(date_range:, organisation:)
+    api_with_long_timeout.content_summary(from: date_range.from, to: date_range.to, organisation: organisation)
+  end
+
 private
 
   def api
     @api ||= GdsApi::ContentDataApi.new
   end
+
+  def api_with_long_timeout
```

Comment:
Is there any reason for not having a long time-out for
all our requests?. It makes sense to me to follow the 
same approach for all our requests.

Reply:
No that should have been removed